app/controllers/web/reminders_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Web::RemindersController < Web::ApplicationController
+  def new
+    @reminder_send = ReminderSendForm.new
+  end
+
+  def create
+    @reminder_send = ReminderSendForm.new(create_params)
+
+    if @reminder_send.invalid?
+      render(:new)
+      return
+    end
```

Comment:
You can fit it all in one line:
`return render(:new) if @reminder_send.invalid?`